troposphere/static/js/models/Instance.js

```diff
@@ -43,6 +43,7 @@ export default Backbone.Model.extend({
             var statusSplit = attrs.status.split(" - ");
             this.set("ip_address", attrs.ip_address);
             this.set("status", attrs.status);
+            this.set("size", attrs.size);
```

Comment:
@steve-gregory There is an issue here with api compatibility. (perhaps its fixed in the corresponding atmo pr). A GET to api v1 for an instance returns a shallow size field namely size_alias, whereas the v2 endpoint for instances returns a not shallow size object. 

`size` is not an attribute that comes back from api/v1